pillar.example

```diff
@@ -35,6 +35,10 @@ cluster:
   # sbd:
   #   # optional: Configure an SBD device
   #   device: /dev/by-label/sbd-disk
+ 
+  # optional: Pre-configured packages installed
```

Comment:
@juadk I like the idea, but I would change the parameter name to: install_packages:
optional: Install required packages to run the cluster

Reply:
@arbulu89 :+1:  You are right, let me do the changes and test them! thanks